hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/RestoreTablesClient.java

```diff
@@ -173,7 +173,8 @@ private void restoreImages(BackupImage[] images, TableName sTable, TableName tTa
     }
 
     if (dirList.isEmpty()) {
-      LOG.warn("Nothing has changed, so there is no need to restore '" + sTable + "'");
+      LOG.info("No incremental changes since full backup for '" + sTable
```

Comment:
nit: also may be "since full" -> "since last full"

Reply:
No objections to parameterized logging, but the remainder of the file also does not use that.

I dislike "since last full", since there might be a more recent full backup that is unrelated to the incremental backup that is being recovered.
Also, to put this into context, there is a single logging message earlier in this method:
```
LOG.info("Restoring '" + sTable + "' to '" + tTable + "' from full" + " backup image "
        + tableBackupPath.toString());
```

So, keeping that in mind, I think it makes sense to just use "full backup".

Reply:
Sure no issues. I am fine with the change as is.